tests/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ if(TARGET demo_sphere_decay)
         set(FILE_REF ${HYDROCHRONO_DATA_DIR}/sphere/postprocessing/sphere_decay_comparison.txt)
 
         add_test(NAME demo_sphere_decay_ref
-                COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/compare.py ${FILE_REF} ${FILE_RST}
```

Comment:
@Huzaifg Do you change ${Python_EXECUTABLE} variable by python3 because there is a problem in your system ?

Reply:
@MaxPal , yes I changed it for my system. Does using python3 not work on your system? I changed it because it fails when you have a Conda environment that has the dependent packages installed (in this case numpy) whereas the base python install does not. This mains that the "site-packages" folder linked to the base python installation is empty and so there is a module not found error.